sample/src/debug/java/com/polidea/cockpit/sample/apipresentation/ApiPresentationContract.kt

```diff
@@ -0,0 +1,18 @@
+package com.polidea.cockpit.sample.apipresentation
+
+interface ApiPresentationContract {
```

Comment:
`ApiPresentation` name is not very intuitive. It's not obvious it means our main screen of the app. We don't call any API in our app. Maybe `MainContract` similarly to our `MainActivity`? Or rename `MainActivity` to `HelloActivity`, `SampleActivity` or something describing our main screen and apply the same naming convention to `...Contract`?